torch/_inductor/fx_passes/mkldnn_fusion.py

```diff
@@ -526,7 +526,7 @@ def _can_be_inplace(_other):
         else:
             return not (
                 isinstance(_other.data, ir.ReinterpretView)
-                or len(_other.get_inputs_that_alias_output()) > 0
+                or len(_other.data.get_inputs_that_alias_output()) > 0
```

Comment:
What's the case when `_other` is a `Buffer`?

Reply:
Modified, thanks!